lib/HotKeysHOC.js

```diff
@@ -1 +1,79 @@
-// TODO: Expose HotKeys functionality as HoC
\ No newline at end of file
+import React, {PureComponent} from 'react';
+import HotKeys from './HotKeys';
+import _ from 'lodash';
+
+const DEFAULT_STATE = {
+  keyboardAction: null,
+  keyboardEvent: null,
+};
+
+/**
+* HotKeysHOC is an HOC that provides the wrappedComponent with the ability to implement keyboard actions
+* without the user wrapping every component with a <HotKeys> component individually
+*
+* See examples/master/HOCWrappedNode.js for an example implementation
```

Comment:
I want to add [JSDoc comments](http://usejsdoc.org/) to the React Hotkeys source code some time in the near future - could you please rewrite this comment to be consistent with the JSDoc standard? 

I'm thinking of the [@example tag](http://usejsdoc.org/tags-example.html) in particular.

Reply:
added! please let me know if i can expand on anything